model/database/registry.yaml

```diff
@@ -127,238 +127,185 @@ groups:
-              brief: "InstantDB"
-              stability: experimental
-            - id: interbase
-              value: "interbase"
-              brief: "InterBase"
-              stability: experimental
+              value: "instantdb"  # no trademark
+              brief: "[InstantDB](https://www.instantdb.com/)"
+              stability: development
             - id: mariadb
-              value: "mariadb"
-              brief: "MariaDB (This value has stability level RELEASE CANDIDATE)"
+              value: "mariadb"  # trademark on 'mariadb'
+              brief: "[MariaDB](https://mariadb.org/)"
               stability: release_candidate
-            - id: maxdb
-              value: "maxdb"
-              brief: "SAP MaxDB"
-              stability: experimental
             - id: memcached
-              value: "memcached"
-              brief: "Memcached"
-              stability: experimental
+              value: "memcached"  # no trademark
+              brief: "[Memcached](https://memcached.org/)"
+              stability: development
             - id: mongodb
-              value: "mongodb"
-              brief: "MongoDB"
-              stability: experimental
+              value: "mongodb"  # trademark on 'mongodb'
+              brief: "[MongoDB](https://www.mongodb.com/)"
+              stability: development
             - id: mssql
-              value: "mssql"
-              brief: "Microsoft SQL Server (This value has stability level RELEASE CANDIDATE)"
+              value: "mssql"  # trademark on 'SQL Server'
+              brief: "[Microsoft SQL Server](https://www.microsoft.com/sql-server)"
               stability: release_candidate
-            - id: mssqlcompact
-              value: "mssqlcompact"
-              deprecated: "Removed, use `other_sql` instead."
-              brief: "Deprecated, Microsoft SQL Server Compact is discontinued."
-              stability: experimental
             - id: mysql
-              value: "mysql"
-              brief: "MySQL (This value has stability level RELEASE CANDIDATE)"
+              value: "mysql"  # trademark on 'mysql'
+              brief: "[MySQL](https://www.mysql.com/)"
               stability: release_candidate
             - id: neo4j
-              value: "neo4j"
-              brief: "Neo4j"
-              stability: experimental
-            - id: netezza
-              value: "netezza"
-              brief: "Netezza"
-              stability: experimental
+              value: "neo4j"  # trademark on 'neo4j'
+              brief: "[Neo4j](https://neo4j.com/)"
+              stability: development
             - id: opensearch
-              value: "opensearch"
-              brief: "OpenSearch"
-              stability: experimental
+              value: "opensearch"  # trademark on 'opensearch'
+              brief: "[OpenSearch](https://opensearch.org/)"
+              stability: development
             - id: oracle
               value: "oracle"
```

Comment:
does this work now that we are sort of considering this to point to a top-level namespace?

or maybe a better question, if we add oracle (database) specific attributes what namespace would they be under? (now that we also have oracle cloud related namespaces I think)

Reply:
renamed to `oracle.db`, but it looks funky along with `couchdb`. Need to sleep on this

It's the case where we have one company and may offerings `oracle.db`, [`oracle.nosql_db`?](https://www.oracle.com/database/nosql/), `oracle.web_logic`, `oracle.jvm` 😅, oracle could is being addedas `oracle_cloud`